packages/manager/modules/nutanix/src/dashboard/component/network-tile/network-tile.controller.js

```diff
@@ -61,6 +53,19 @@ export default class NutanixNetworkTileController {
       .finally(() => {
         this.loadingUpgradeOptions = false;
       });
+    this.setPrivateBandwidth();
+  }
+
+  setPrivateBandwidth() {
+    this.privatebandwidth = {
+      value:
+        parseInt(
+          this.privateBandwidthPlanCode
+            .split('-')
+            .find((ele) => /^\d+$/.test(ele)),
+        ) / 1000,
+      unit: 'Gbps',
```

Comment:
is this displayed on UI, if so you need translate it

Reply:
It is an input to the filter 'serverBandwidth'. And the filter can handle only 'Gbps', 'Mbps' and so on.
Unfortunately, the filter doesn't translate as well.